packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -295,11 +295,40 @@ export let DataSpreadsheet = React.forwardRef(
       [cellEditorValue, onDataUpdate]
     );
 
+    const getHeaderKeyText = useCallback((element: any) => {
+      if (!element || typeof element !== 'object') {
+        return null;
+      }
+
+      if (element.type === 'span' && element.props.headerKey === true) {
```

Comment:
How come you check explicitly for `span` here?

Reply:
I was following how we had our setup with, will change this out so it isn't specific, thanks!